plugins/python/Makefile.in

```diff
@@ -44,8 +44,10 @@ INSTALL_BACKUP = @INSTALL_BACKUP@
 LT_LIBS = $(top_builddir)/lib/util/libsudo_util.la
 LIBS = $(LT_LIBS)
 
+LIBPYTHONPLUGIN = python_plugin.la
+
 # C preprocessor flags
-CPPFLAGS = -I$(incdir) -I$(top_builddir) -I$(top_srcdir) @CPPFLAGS@ @PYTHON_INCLUDE@
+CPPFLAGS = -I$(incdir) -I$(top_builddir) -I$(top_srcdir) -DSRC_DIR=\"$(abspath $(srcdir))\" @CPPFLAGS@ @PYTHON_INCLUDE@
```

Comment:
Using $(abspath) creates a dependency on GNU make.  Currently sudo will build with any version of make and I'd like it to stay that way.  You can @abs_srcdir@ from configure instead by defining abs_srcdir in Makefile.in near where srcdir is defined.  E.g.

```
srcdir = @srcdir@
devdir = @devdir@
top_builddir = @top_builddir@
top_srcdir = @top_srcdir@
abs_srcdir = @abs_srcdir@

...

CPPFLAGS = -I$(incdir) -I$(top_builddir) -I$(top_srcdir) -DSRC_DIR=\"$(abs_srcdir)\" @CPPFLAGS@ @PYTHON_INCLUDE@
```

```

Reply:
This can be changed as part of a larger diff to use the absolute versions of src and build dirs where needed.